README.md

```diff
@@ -2,197 +2,74 @@
-
-Once you have the necessary prerequisites, LiT can be compiled by running the following
-commands:
+Download the latest binaries from the [releases](https://github.com/lightninglabs/lightning-terminal/releases) page. You can also download the latest binaries from the command line:
 
 ```
-git clone https://github.com/lightninglabs/lightning-terminal.git
-cd lightning-terminal
-make install
+TARBALL="lightning-terminal-linux-amd64-v0.2.0-alpha"
+BASE="https://github.com/lightninglabs/lightning-terminal/releases/latest/download"
+wget -q $BASE/$TARBALL.tar.gz -O - | tar -xz && cd $TARBALL
 ```
 
-This will produce the `litd` executable and add it to your `GOPATH`. The CLI binaries for
-`lncli`, `loop`, and `frcli` are not created by `make install`. You will need to download
-those binaries from the [lnd](https://github.com/lightningnetwork/lnd/releases),
-[loop](https://github.com/lightninglabs/loop/releases), and
-[faraday](https://github.com/lightninglabs/faraday/releases) repos manually.
-
-#### Executing CLI Commands
-
-When executing `loop` and `frcli` commands, you will need to specify the connection info
-since the daemons are now integrated into `lnd`'s GRPC server.
-
-Examples:
-
-```
-loop --rpcserver=localhost:10009 --tlscertpath=$HOME/.lnd/tls.cert --macaroonpath=$HOME/.lnd/data/chain/bitcoin/mainnet/admin.macaroon
-```
+## Execution
+Run Lightning Terminal with a local LND instance:
 
 ```
-frcli --rpcserver=localhost:10009 --tlscertpath=$HOME/.lnd/tls.cert --macaroonpath=$HOME/.lnd/data/chain/bitcoin/mainnet/admin.macaroon
+./litd --uipassword=UP48lm4VjqxmOxB9X9stry6VTKBRQI
 ```
 
-## Configuration
-
-LiT only has a few configuration parameters itself.
-
-#### Required
+Visit https://localhost:8443 to access Terminal.
 
-You must set `httpslisten` to the host & port that the https server should listen on. Also
-set `uipassword` to a strong password to use to login to the website in your browser. A
-minimum of 8 characters is required. In a production environment, it's recommended that
-you store this password as an environment variable.
+Note that a password with a minimum of 8 characters is required to run Lightning Terminal. In a production environment, it's recommended that you store this password as an environment variable.
 
-#### Optional
+To use LiT with a remote LND instance please [follow these instructions](./doc/remote.md). 
 
-You can also configure the HTTP server to automatically install a free SSL certificate
-provided by [Let's Encrypt](https://letsencrypt.org/). This is recommended if you plan to
-access the website from a remote computer and do not want to deal with the browser warning
-you about the self-signed certificate. View the
+## Configuration
+If you plan to access the website from a remote computer and don’t want to deal with the browser warning you about the self-signed certificate, you can configure the HTTP server to use a certificate from [Let's Encrypt](https://letsencrypt.org/). View the
```

Comment:
I don't think this info needs to be right here in the main README. It can get moved to the config docs for remote & integrated modes.